### ISSUE TITLE
Why the data of flecs and mustache sometimes looks so strange where the number of entites is small?

### ISSUE THREAD

**Issue by fexla** [abeimler/ecs_benchmark#25]

**Comment by abeimler** [comment]
The "number" property looks a bit off, cause the measurements are in **time per entity** and not "time for the whole update (all entities)".

You can have a look at the benchmark runs, with the execution time for the N entities:

## mustache

    Benchmark                                      Time             CPU   Iterations UserCounters...

![image](https://github.com/abeimler/ecs_benchmark/assets/22779737/3223fec5-03cf-445f-b0be-d94c9202c25e)


## flecs

    Benchmark                                      Time             CPU   Iterations UserCounters...

![image](https://github.com/abeimler/ecs_benchmark/assets/22779737/fa5a15c1-3545-40da-aa23-f6cbea46e7e5)



Basically you get more from the frameworks if you are using a good amount of entities.

I assume the performance lost done by the overhead of the framework (per entity).

**Comment by abeimler** [comment]
I assume the number looks a bit of, cause the measurements in the graph are in **"time per entity"** and not "time for the whole update (all entities)".

If you look at the benchmarks, you see the time is in time for all entities (the whole operation):

## mustache

![image](https://github.com/abeimler/ecs_benchmark/assets/22779737/77adeb1e-097e-45f7-b76a-a3f8d4798366)


## flecs

![image](https://github.com/abeimler/ecs_benchmark/assets/22779737/7d27f9f5-0692-4f99-81c9-e83e7e9c3ceb)



Basically you get more from the frameworks, if you are using the right/good amount of entities.

_Also, maybe the histogram looks a bit dramatic, cause I average the numbers of the group.._
